src/Stryker.RegexMutators/Stryker.RegexMutators/RegexMutation.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RegexParser.Nodes;
+
+namespace Stryker.RegexMutators
+{
+    public class RegexMutation
+    {
+        public RegexNode OriginalNode { get; set; }
+        public RegexNode ReplacementNode { get; set; }
+        public string Pattern { get; set; }
```

Comment:
> Is this the original pattern or the replacement pattern? And is this needed if we have the nodes as well?

This is the replacement pattern. It is needed because this is the actual mutated string. Maybe we could do without the nodes.